### PR TITLE
save last tab index in upload request group view (#560)

### PR DESCRIPTION
**Pull request by imGok**
sabtab

### CODE REVIEW

File: lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart [hunk 1]
```diff
@@ -58,6 +58,8 @@ class UploadRequestGroupViewModel extends BaseViewModel {
   List<UploadRequestGroup> _uploadRequestListActiveClosed = [];
   List<UploadRequestGroup> _uploadRequestListArchived = [];
 
+  int _tabIndex = 0;
```
Comment (hoangdat) [inline lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart:61]: why we need store here? How about store it directly in UI state?

Comment (imGok) [reply lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart:61]: I thought dispatching an action at every tab switch was a bad idea and useless wdyt ?

Comment (hoangdat) [reply lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart:70]: yes, it should be consistent for all app. Not too many places to store the state of the application.

**Review by huynguyennovem — APPROVED** [review]
\lgtm

**Pull request merged**